plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/StartMojo.java

```diff
@@ -328,7 +329,7 @@ protected SwarmExecutor executor(final Path appPath, final String name,
                         Files.createTempFile(Paths.get(this.projectBuildDir), "swarm-", "-cp.txt").toFile() :
                         Files.createTempFile("swarm-", "-cp.txt").toFile();
 
-                tmp.deleteOnExit();
+                getPluginContext().put("swarm-cp-file", tmp);
```

Comment:
Actually I'm thinking here... this is OK for the `start` goal, because there should always be a corresponding `stop`, but what about `run`? `RunMojo extends StartMojo`, so if I do `mvn wildlfy-swarm:run` and then Ctrl+C that process, the temp file will stay on the filesystem. Right?